### DISCUSSION TITLE
No xcodebuild speedup on macos-13-xl image

### DISCUSSION THREAD

**Discussion by asendra** [actions/virtual-environments#8217] [General questions]
We have a quite standard build process for our app using Fastlane, and we wanted to try the "new" `macos-13-xl` image to see how much improvement we could get compared to the regular `macos-13` image we are using now.

Quite shockingly, the times for our xcodebuild commands are taking the same amount of time. At least the same time as our best runs in a regular `macos-13` image, which is a bit variable, and sometimes takes a bit more time. 
Is there something we should check or verify to make sure that we are actually getting the XL runners?

I've even tried to manually specify the number of jobs, which works locally on a mac, (`xcodebuild .... -jobs 12`), and no change whatsoever.

**Comment by mikhailkoliada** [comment]
Hello! 

the macOS-xl runners specs are:

* 12-core CPU (x86_64)
* 30 GB of RAM
* 14 GB of SSD space

but as we are unfamiliar with your code base / project specifics / runner resources utilisation we may not judge if it is something runner-specific or not. Plain increase of number of parallel jobs may indeed not have desired effect.
